Remotion/Web/UnitTests/Core/Security/HttpContextPrincipalProviderTest.cs

```diff
@@ -31,40 +32,42 @@ public class HttpContextPrincipalProviderTest
     [Test]
     public void GetPrincipal_HttpContextUserIsNotAuthenticated_ReturnsNullSecurityPrincipal ()
     {
-      var identityStub = MockRepository.GenerateStub<IIdentity>();
-      identityStub.Stub (_ => _.IsAuthenticated).Return (false);
+      var identityStub = new Mock<IIdentity>();
+      identityStub.Setup (_ => _.IsAuthenticated).Returns (false);
 
-      var principalStub = MockRepository.GenerateStub<IPrincipal>();
-      principalStub.Stub (_ => _.Identity).Return (identityStub);
+      var principalStub = new Mock<IPrincipal>();
+      principalStub.Setup (_ => _.Identity).Returns (identityStub.Object);
 
-      var httpContextStub = MockRepository.GenerateStub<HttpContextBase>();
-      httpContextStub.User = principalStub;
+      var httpContextStub = new Mock<HttpContextBase>();
+      httpContextStub.SetupAllProperties();
```

Comment:
Hmm as this is a manual change anyway, I think it might be better to use `.SetupProperty(...)` instead, to be more specific and show that properties aren't set up at other points in the call chain. (multiple occurrences)

@MichaelKetting 's opinion would be interesting here.